src/classes/music_album.rb

```diff
@@ -0,0 +1,13 @@
+class MusicAlbum
```

Comment:
Hi @[brytebee](https://github.com/brytebee),

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Required Changes 
![giphy](https://user-images.githubusercontent.com/93735137/163804540-882ea424-dc9a-4572-b774-1f6dd63ef0c7.gif)

Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

N/A
Cheers and Happy 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.